stardis/radiation_field/base.py

```diff
@@ -0,0 +1,32 @@
+from stardis.radiation_field.opacities import Opacities
+import numpy as np
+
+
+class RadiationField:
+    """
+    Class containing information about the radiation field.
+    ###TODO Radiation field temperature should be a separate attribute, for the case of differing gas and radiation.
```

Comment:
It does not appear to be an attribute right now, so is this done?

Reply:
It's not. I can add it fairly easily, but I figured I would hold off on until there was a reason to include it. The model temperature which gets read in is currently used for everything because all the temperatures in our stars are coupled. Do you think it's worth adding now even if it's not hooked up to anything?

Reply:
No, probably not